pkg/cli/idle/idle.go

```diff
@@ -268,6 +268,9 @@ func (o *IdleOptions) calculateIdlableAnnotationsByService(infoVisitor func(reso
 		}
 		// just get the unversioned version of this
 		gk := schema.GroupKind{Group: gv.Group, Kind: ref.Kind}
+		if gk.Kind == "StatefulSet" {
```

Comment:
I'd suggest being more accurate ie.:
```golang
if kappsv1.SchemeGroupVersion.WithKind("StatefulSet").GroupKind() == gk {
...
```
this will ensure that only k8s apps StatefulSet will be ignored.